app/controllers/show-problem.js

```diff
@@ -50,119 +51,18 @@ define(['./module'], function(controllers){
-                        UserService.setSaveChangeStatus(false);
-                    }
-                });
-                $scope.$watch('problem.Title', function(newValue, oldValue) {
-                    if(newValue != oldValue ){
-                        $scope.editStatusClass =  adminToShowProblemService.getEditStatus(1);
-                        $scope.problem.Title = newValue;
-                        UserService.setSaveChangeStatus(false);
-                    }
-                });
-                $scope.$watch('problem.Severity', function(newValue, oldValue) {
-                    if(newValue != oldValue ) {
-                        $scope.editStatusClass =  adminToShowProblemService.getEditStatus(1);
-                        $scope.problem.Severity = newValue;
-                        UserService.setSaveChangeStatus(false);
 
-                    }
-                });
-                $scope.$watch('problem.Content', function(newValue, oldValue) {
-                    if(newValue != oldValue && UserService.getSaveChangeStatus() == true) {
-                        $scope.problem.Content = newValue;
-                        UserService.setSaveChangeStatus(false);
-                        $scope.editStatusClass =  adminToShowProblemService.getEditStatus(1);
-                    }
-                });
-                $scope.$watch('problem.Proposal', function(newValue, oldValue) {
-                    if(newValue != oldValue && UserService.getSaveChangeStatus() == true) {
-                        $scope.problem.Proposal = newValue;
-                        UserService.setSaveChangeStatus(false);
-                        $scope.editStatusClass =  adminToShowProblemService.getEditStatus(1);
-                    }
-                });
-            }
-        })
-            .error(function (data, status, headers, config) {
-                throw error;
-            });
-        */
-        //end get problem request info
-
-
-        $scope.dropzoneConfig = {
-            'options':{autoProcessQueue:false,
-                url:'/api/photo/'+$routeParams.problemID,
-                method:"POST",
-                uploadMultiple:true,
-                maxFilesize:10,
-                parallelUploads:10,
-                thumbnailWidth:100,
-                thumbnailHeight:100,
-                acceptedFiles:'.jpg,.jpeg',
-               // dictFileTooBig: "Файл великого розміру ({{filesize}}MB). Максимальний розмір файлу: {{maxFilesize}}MB.",
-
-
-                //dictInvalidFileType:"Невірний формат файлу. Допустимі формати : jpg,jpeg",
-                clickable:".previews,.dropFieldForShowProblem",
-
-                previewsContainer:".previews"
-                //w
+        ProblemService.getProblemPhotos($routeParams.problemID).success(function (data) {
+            $scope.photos = data;
+        });
 
-            }
-        };
         $scope.showDrop = function(){
             if($scope.showDropField==false)
```

Comment:
scope.x = 0.3 is an opacity from activity.html template. we don't need that panel at all